cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/DataRecoveryCommand.java

```diff
@@ -50,4 +79,215 @@ StorageFactory createStorageFactory(ScheduledExecutorService executorService) {
         return loader.load(configSetupHelper, getServiceConfig().getStorageImplementation(),
                 getServiceConfig().getStorageLayout(), executorService);
     }
+
```

Comment:
All the stuff you put in here will be available in all other commands, when it is not needed. If `DurableLogInspectCommand` shared functionality with `DurableLogRepairCommand`, then make `DurableLogInspectCommand` extend `DurableLogRepairCommand and mark the relevant methods as `protected`. I think this is better than putting all this code, which is specific, in a general class to all Admi ncommands.

Reply:
removed unrelated changes.